### PR TITLE
Fixing missing comma to allow for additional entries without breaking…

### PR DESCRIPTION
**Pull request by rasaha91**
… the build

#### Please select one of the following
- [ ] I am removing an existing difference between facebook/react-native and microsoft/react-native-macos :thumbsup:
- [ ] I am cherry-picking a change from Facebook's react-native into microsoft/react-native-macos :thumbsup:
- [ ] I am making a fix / change for the macOS implementation of react-native
- [x] I am making a change required for Microsoft usage of react-native

## Summary

This change updates a patch for android to prevent a build error when additional entries are added.

## Changelog

[Android] [Fixed] - Fix patching to avoid build error

## Test Plan

Built locally after working through unrelated build breaks in main and was able to build successfully. Will cherry-pick the changes to 0.64-stable afterwards.

### CODE REVIEW

**Comment by rasaha91** [comment]
Failure in Android PR is due to a bad merge from 0.66. I fixed that locally and verified a successful build. Leaving the actual fix to those working on 0.66 since I'm not sure what the appropriate merge should be (I resolved the conflict by keeping both changes for the sake of testing the build).

**Comment by kelset** [comment]
Could you clarify what you mean with
> I fixed that locally and verified a successful build. Leaving the actual fix to those working on 0.66 since I'm not sure what the appropriate merge should be (I resolved the conflict by keeping both changes for the sake of testing the build).

My understanding was that the fix you are committing here in this PR is the same one that @HeyImChris did as part of this PR https://github.com/microsoft/react-native-macos/pull/989/files to fix the Android pipeline. But by itself is not enough to fix the Android CI? 🤔

What other fixes are needed?

**Comment by rasaha91** [comment]
The android PR is breaking because there was an unresolved merge conflict in ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java

File: ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java [hunk 1]
```diff
@@ -828,16 +828,13 @@ public void setKeyboardType(ReactEditText view, @Nullable String keyboardType) {
       // This will supercede secureTextEntry={false}. If it doesn't, due to the way
       //  the flags work out, the underlying field will end up a URI-type field.
       flagsToSet = InputType.TYPE_TEXT_VARIATION_VISIBLE_PASSWORD;
-<<<<<<< HEAD
     } else if ((view.getStagedInputType() & AUTOCAPITALIZE_FLAGS) != 0) {
       // This prevents KEYBOARD_TYPE_FLAGS from being unset when the keyboardType is
       // default, null, or unsupported, and autocapitalize is on.
       // Unsetting these flags breaks the autoCapitalize functionality.
       unsettingFlagsBreaksAutocomplete = true;
```
Comment (kelset) [inline ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java:835]: I think you need to remove this section to match how things have been done in 0.66:
Screenshot 2022-02-01 at 16 54 29

Comment (kelset) [reply ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java:835]:
```suggestion
```

**Review by kelset — APPROVED** [review]

**Review by HeyImChris — APPROVED** [review]

**Pull request merged**